tomviz/OperatorPropertiesPanel.cxx

```diff
@@ -17,15 +17,19 @@
 
 #include "ActiveObjects.h"
 #include "DataSource.h"
+#include "EditOperatorDialog.h"
 #include "Operator.h"
 #include "OperatorWidget.h"
 #include "Pipeline.h"
 #include "Utilities.h"
 
+#include "pqCoreUtilities.h"
```

Comment:
Is this needed? If so it should use <> rather than ""

Reply:
*tries taking it out*... apparently not.  We changed to use the new Utilities function (tomviz::mainWidget) from pqCorUtilities::mainWidget when we rebased onto the new pipeline changes and I guess we forgot to get rid of this include.  I've added a commit to remove this.